svelte.config.js

```diff
@@ -11,6 +11,10 @@ const config = {
   kit: {
     adapter: process.env.PREVIEW ? node() : auto(),
   },
+  package: {
+    dir: 'svelte-package',
+    extensions: ['.ts', '.svelte'],
```

Comment:
Is this `.ts` item still needed here? I think this was one of our failed attempts at a fix.